ansible/roles/service_authz/tasks/main.yml

```diff
@@ -1,6 +1,63 @@
 ---
 # tasks file for service-authz
-- name: work with docker
+- name: Create folders and copy files
+  block:
+    - name: Create a directory for nginx logs
+      ansible.builtin.file:
+        path: /var/log/nginx_logs
+        state: directory
+        mode: '0755'
+    - name: Create a directory for certbot logs
+      ansible.builtin.file:
+        path: /var/log/certbot_logs
+        state: directory
+        mode: '0755'
+    - name: Create a directory for certbot storage
+      ansible.builtin.file:
+        path: /etc/letsencrypt
+        state: directory
+        mode: '0755'
+    - name: Create a directory for SSL certificate (certbot)
+      ansible.builtin.file:
+        path: /etc/letsencrypt/live/esa.authz.smodata.net
+        state: directory
+        mode: '0755'
+    - name: Copy file certbot.sh
```

Comment:
So we are using a separate container for letsencrypt. Why not add certbot.sh to our container image in Dockerfile,certbot